.gitignore

```diff
@@ -34,3 +34,6 @@ storybook-static
 # ESlint Cache Files
 .eslintmdcache
 .eslintjscache
+
+# Editor
+.idea
```

Comment:
Editor Specific files should not be part of a `.gitignore` file. You should probably add  a `.gitignore` file on your `~` (home folder)